### DISCUSSION TITLE
Temu OTP code

### DISCUSSION THREAD

**Discussion by vdias** [dani-garcia/vaultwarden#4736] [Q&A]
First time that i face problems with OTP codes...

Trying to set OTP code for temu.com, but all code generated are reported as incorrect... anyone face this same problem?

**Comment by ItzmeSwapy** [comment]
did you find the solution to this?

**Comment by Masgalor** [comment]
I don't use temu, so I can't test this myself, but the following questions come to mind.

Maybe they have a parameter in their URI that is not understood by bitwarden, can you post an example?
Does it work with other otp apps, e.g. Google Authenticator?

**Comment by BlackDex** [comment]
Are you sure the date/time is correct?
And indeed, check the OTP entry for some weird items maybe.
